### PR TITLE
Deploy RBAC for prometheus-k8s SA OSD-6308

### PR DESCRIPTION
**Pull request by csheremeta**

### CODE REVIEW

File: hack/generate-operator-bundle.py [hunk 1]
```diff
@@ -68,6 +69,13 @@
         }
     )
 
+# Copy all prometheus yaml files over to the bundle output dir:
+prom_files = [ f for f in os.listdir('deploy') if re.match(r'.*prometheus-k8s.*.yaml', f) ]
```
Comment (2uasimojo) [inline hack/generate-operator-bundle.py:73]: nits:
- If you use `re.search`, you don't need the initial `.*` -- it doesn't use implicit anchors.
- The last `.` will theoretically match any character. If you were being precise, you should use `\.`.

File: hack/generate-operator-bundle.py [hunk 2]
```diff
@@ -68,6 +69,13 @@
...
+for file_name in prom_files:
+    full_path = os.path.join('deploy', file_name)
+    if (os.path.isfile(os.path.join('deploy', file_name))):
```
Comment (2uasimojo) [inline hack/generate-operator-bundle.py:76]:
```suggestion
    if (os.path.isfile(full_path)):
```

**Review by 2uasimojo — APPROVED** [review]
This makes sense. A couple of nits if you want to fix. Otherwise feel free to unhold.

/lgtm
/approve
/hold

**Comment by csheremeta** [comment]
@2uasimojo thank you for the coding tips! :) changes have been implemented

**Review by fahlmant — APPROVED** [review]

**Pull request merged**